plenum/test/txn_author_agreement/test_state_proof_for_get_taa.py

```diff
@@ -0,0 +1,33 @@
+import base64
+
+import base58
+
+from plenum.common.constants import REPLY, STATE_PROOF, ROOT_HASH, PROOF_NODES
+from plenum.common.util import randomString
+from plenum.test.txn_author_agreement.helper import sdk_send_txn_author_agreement, sdk_get_txn_author_agreement, \
+    check_valid_proof, taa_digest
+from state.pruning_state import PruningState
+
+
+# TODO: Change txnPoolNodeSet to nodeSetWithOneNodeResponding after
+#  correct state proof checking is implemented in Indy SDK
+def test_state_proof_returned_for_get_txn_author_agreement(
```

Comment:
I think we need to have multiple tests (and hence multiple `sdk_get_txn_author_agreement`) for each variant of GET_TAA: latest, by digest, by ts.

Reply:
Of course they are planned. Also tests checking for consistency proofs for absent keys are planned as well.